packages/react-components/src/organisms/InterestGroupCard.tsx

```diff
@@ -53,12 +59,19 @@ const InterestGroupCard: React.FC<InterestGroupCardProps> = ({
       </LinkHeadline>
       {!active && <StateTag icon={inactiveBadgeIcon} label="Inactive" />}
     </div>
+    {googleDrive && (
+      <Link href={googleDrive} buttonStyle small>
+        {googleDriveIcon} Access Drive
+      </Link>
+    )}
     <Anchor
       href={
         network({}).interestGroups({}).interestGroup({ interestGroupId: id }).$
       }
     >
-      <Paragraph accent="lead">{description}</Paragraph>
+      <Paragraph accent="lead" styles={descriptionStyles}>
+        {description}
+      </Paragraph>
     </Anchor>
     <TagList min={2} max={3} tags={tags} />
     {active && (
```

Comment:
And it seems we display the number of teams even when the interest group is inactive
![Screenshot 2024-02-14 at 16 36 38](https://github.com/yldio/asap-hub/assets/16595804/98e106b2-c44b-4ab3-987c-fef5c804ff9e)